sass/theme/wrbb-styles/album-review.scss

```diff
@@ -0,0 +1,111 @@
+#header-line {
```

Comment:
This probably wasn't explained to you, but Gulp takes all of the styles from every SCSS file that we write and combines them into one, so if I add a style to `interview.scss` for example, it will be applied to every page. So many of these selectors are redundant, can you remove them?